WooCommerce/Classes/Extensions/UITableViewCell+Helpers.swift

```diff
@@ -18,6 +18,13 @@ extension UITableViewCell {
         backgroundColor = .listForeground(modal: false)
     }
 
+    /// Applies the default selected background color
+    ///
+    func applyDefaultSelectedBackgroundStyle() {
+        selectedBackgroundView = UIView()
+        selectedBackgroundView?.backgroundColor = .listSelectedBackground
+    }
```

Comment:
I like this being in an extension.

What about using `UIBackgroundConfiguration` as in this PR? #11691

It may add some issues with iOS 15 (see #11786 where we fixed one) but they _should_ be something we can handle, and it's a more recent way to do it.